ush/machine/wcoss2.yaml_nrt

```diff
@@ -0,0 +1,57 @@
+  METPLUS_PATH: /apps/ops/para/libs/intel/19.1.3.304/metplus/4.1.1
+  MET_BIN_EXEC: bin
+  MET_INSTALL_DIR: /apps/ops/para/libs/intel/19.1.3.304/met/10.1.1
+  DOMAIN_PREGEN_BASEDIR: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/FV3LAM_pregen
+  QUEUE_DEFAULT: devhigh
+  QUEUE_FCST: devhigh
+  QUEUE_HPSS: dev_transfer
+  RUN_CMD_FCST: mpiexec -n ${PE_MEMBER01} -ppn ${PPN_RUN_FCST} --cpu-bind core -depth ${OMP_NUM_THREADS_RUN_FCST}
+  RUN_CMD_POST: mpiexec -n ${nprocs}
+  RUN_CMD_PRDGEN: mpiexec -n ${nprocs} --cpu-bind core cfp
+  RUN_CMD_SERIAL: time
+  RUN_CMD_UTILS: mpiexec -n ${nprocs}
+  RUN_CMD_AQM: mpiexec -n ${nprocs} -ppn ${ppn_run_aqm} --cpu-bind core -depth ${omp_num_threads_run_aqm}
+  RUN_CMD_AQMLBC: mpiexec -n ${NUMTS}
+  SCHED_NATIVE_CMD: -l place=excl
+  PRE_TASK_CMDS: '{ ulimit -s unlimited; ulimit -a; }'
+  TEST_EXTRN_MDL_SOURCE_BASEDIR: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data
+  TEST_PREGEN_BASEDIR: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/FV3LAM_pregen
+  FIXaer: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/fix/fix_aer
+  FIXgsm: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/fix/fix_am
+  FIXlut: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/fix/fix_lut
+  FIXorg: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/fix/fix_orog
+  FIXsfc: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/fix/fix_sfc_climo
+  FIXshp: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/NaturalEarth
+  EXTRN_MDL_DATA_STORES: hpss aws nomads
+data:
+  ics_lbcs:
+    GSMGFS: compath.py ${envir}/gsmgfs/${gsmgfs_ver}/gsmgfs.${PDYext}
+    FV3GFS: compath.py ${envir}/gfs/${gfs_ver}/gfs.${PDYext}/${cycext}/atmos
+    RAP: compath.py ${envir}/rap/${rap_ver}/rap.${PDYext}
+    NAM: compath.py ${envir}/nam/${nam_ver}/nam.${PDYext}
+    HRRR: compath.py ${envir}/hrrr/${hrrr_ver}/hrrr.${PDYext}/conus
+cpl_aqm_parm:
+  AQM_CONFIG_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/aqm/epa/data
+  DCOMINbio_dfv: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/aqm/bio
+  DCOMINdust_dfv: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/FENGSHA
+  DCOMINcanopy_dfv: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/canopy
+  #DCOMINfire_dfv: /lfs/h1/ops/dev/dcom 
+  #DCOMINfire_dfv: /lfs/h2/emc/physics/noscrub/jianping.huang/RAVE_fire/RAVE_NA_NRT
+  DCOMINfire_dfv: /lfs/h2/emc/physics/noscrub/kai.wang/RAVE_fire/RAVE_NA_NRT/RAVE_raw_scaleup 
+  DCOMINchem_lbcs_dfv: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/LBCS/AQM_NA13km_AM4_v1
+  DCOMINgefs_dfv: /lfs/h1/ops/prod/com/gefs/v12.3 
+  DCOMINpt_src_dfv: /lfs/h2/emc/physics/noscrub/Youhua.Tang/nei2016v1-pt/v2023-01-PT
+  DCOMINairnow_dfv: /lfs/h1/ops/prod/dcom
+  COMINbicor: /lfs/h2/emc/physics/noscrub/jianping.huang/Bias_correction/aqmv7.0.84
+  COMOUTbicor: /lfs/h2/emc/physics/noscrub/jianping.huang/Bias_correction/aqmv7.0.84
+  NEXUS_INPUT_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/nexus_emissions
+  NEXUS_FIX_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/nexus/fix
+  NEXUS_GFS_SFC_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/GFS_DATA
+
```

Comment:
@JianpingHuang-NOAA, why do you need this file? What is the difference from `wcoss2.yaml`?